nova_vm/src/ecmascript/builtins/array.rs

```diff
@@ -80,6 +80,17 @@ impl Array {
         agent[self].elements.is_trivial(agent)
     }
 
+    pub(crate) fn shallow_clone(self, agent: &mut Agent) -> Array {
```

Comment:
**suggestion:** This could maybe be renamed to "clone_array" (a nod to us only taking the array features) or "to_cloned" (a nod to the prototype methods that use this).

Reply:
I opted for `to_cloned` in order to make it more readable when calling `array.to_cloned()`